src/zepto.js

```diff
@@ -716,7 +713,7 @@ var Zepto = (function() {
         var element = this[0]
         if (typeof property == 'string') {
           if (!element) return
-          return element.style[camelize(property)] || getComputedStyle(element, '').getPropertyValue(property)
+          return element.style[camelize(property)] || getComputedStyle(element, '').getPropertyValue(dasherize(property))
```

Comment:
This is a feature change. Ideally, a code cleanup and an unrelated feature change should be submitted in separate PRs. However, for now it's fine. Please add a test with what this accomplishes. Thanks!